docs/INSTALL.rst

```diff
@@ -70,24 +70,24 @@ Copy the following lines in file ``hil_env``
 
 Before starting this procedure do::
 
-        source hil_env
+        $ source hil_env
 
 
 Setting up system user for HIL:
 --------------------------------
 
 First create a system user ``${HIL_USER}`` with::
 
-  sudo useradd --system ${HIL_USER} -d /var/lib/hil -m -r
+  $ sudo useradd --system ${HIL_USER} -d /var/lib/hil -m -r
 
 
 The HIL software itself can then be installed as root by running::
 
-    sudo su -
-    cd /root
-    git clone https://github.com/CCI-MOC/hil
-    cd hil
-    sudo python setup.py install
+    $ sudo su -
+    $ cd /root
+    $ git clone https://github.com/CCI-MOC/hil
+    $ cd hil
+    $ sudo python setup.py install
```

Comment:
if you are root at this point, you don't need to sudo.

Reply:
true. But how could you even `cd /root` if you a not a root user? hmm.

Reply:
nvm. `sudo su-`

Reply:
Related to the convention discussion above, A common convention is to display root prompts with `#` rather than `$`, which I think is a good idea if we go with the prompt indicators.